hack/jenkins/e2e.sh

```diff
@@ -124,6 +124,21 @@ GCE_PARALLEL_FLAKY_TESTS=(
     "Services.*release.*load\sbalancer"
     )
 
+GCE_SOAK_CONTINUOUS_SKIP=(
```

Comment:
might also be worth documenting somewhere how this list was generated, since it's subtly different from the other skip lists.